Deploy-CAM.ps1

```diff
@@ -2648,7 +2648,7 @@ function Confirm-ModuleVersion()
     $AzureRMModule = Get-Module -ListAvailable -Name "AzureRM"
     if ( $AzureRMModule ) {
         # have an AzureRM version - check that.
-        if ( [version]$AzureRMModule.Version.ToString() -lt [version]$MinAzureRMVersion) {
```

Comment:
how about there is only one version. Is it still an array?

Reply:
Yes, it returns an object and will always have an index 0, currently works perfectly fine with one version installed. 

All I have seen, found and tested around has had the list sorted with the newest on top. But I can't confirm this is the design, no documentation saying this is the case. I can definitely implement a sort with a couple lines, should I do this?

Reply:
@rohml yeah in my experience if something magically works and isn't by any design then it often magically unworks at an inopportune time in the future. So if it's easy to sort, I'd suggest we do that. Thanks!